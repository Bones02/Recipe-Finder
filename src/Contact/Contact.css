.contact {
    text-align: center;
    margin-bottom: 60px;
    margin-top: 60px;
}