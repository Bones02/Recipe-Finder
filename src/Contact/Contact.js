import React, { Component } from "react";
import './Contact.css'
 
class Contact extends Component {
  render() {
    return (
      <div className="contact">
        <h2>HAVE QUESTIONS?</h2>
        <p>Email me at anbonesteel@gmail.com</p> 
        <a target="_blank" href="https://www.linkedin.com/in/alyssabonesteel/">Connect with me on LinkedIn</a>
      </div>
    );
  }
}
 
export default Contact;