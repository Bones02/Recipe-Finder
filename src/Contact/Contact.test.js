import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Contact from './Contact'

describe(`Contact component`, () => {
  it('renders a .contact by default', () => {
    const wrapper = shallow(<Contact />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})