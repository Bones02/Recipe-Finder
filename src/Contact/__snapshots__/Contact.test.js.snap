// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component renders a .contact by default 1`] = `
<div
  className="contact"
>
  <h2>
    HAVE QUESTIONS?
  </h2>
  <p>
    The easiest thing to do is view the curriculum at
    <a
      href="thinkfl.com"
      target="_blank"
    >
       Thinkful
    </a>
    .
  </p>
</div>
`;
